wiki/Article_Styling_Criteria/en.md

```diff
@@ -398,13 +421,21 @@ For example use:
 
 ### osu!
 
-The name of the game `osu!` is **never capitalised**, even if it is the first word in the sentence.
+The name of the game, osu!, is **never capitalised nor italicised**.
 
-The osu! official branding are to not use any spaces. Examples may include:
+The osu! official branding are to not use any spaces. Examples include:
 
-- `osu!taiko`
-- `osu!mania`
+- `osu!academy`
+- `osu!api`
+- `osu!catch`
 - `osu!direct`
+- `osu!mania`
+- `osu!store`
+- `osu!stream`
+- `osu!supporter`
+- `osu!taiko`
+- `osu!talk`
+- `osu!tourney`
 
 Other brandings that aren't covered by the osu! official branding must use spaces. Examples may include:

```

Comment:
this section's a little odd, since we have official references to osu!community, osu!wiki, etc

i'd say you can use these sort of things to mention the general "object" of its particular focus - the osu!community being the osu! community at large, the osu!wiki being the osu! wiki, so on, so forth - but only official projects like the osu!talk bear this sort of stuff officially speaking, if that makes sense

bit convoluted though

Reply:
If you backlog with this search query `from:ppy#0001 in:osu-wiki branding` and find ppy's message on 07/12/2017, peppy said that official branding has no space, while anything should have one. However, throughout the website (that aren't wiki or forums), this is heavily inconsistent at times 🤷‍♂️

Reply:
so should I move `osu! community` and `osu! wiki` to the other list to be `osu!community` and `osu!wiki`?

Reply:
i guess the name of the game here is to keep consistent with what the project lead wants - though i'd say the osu!wiki is the osu!wiki and not the osu! wiki, since any random wikia could also be a "osu! wiki", the community one we can probably use the space with though

beyond the scope of this otherwise comprehensive PR though so i'm not gonna hold it up any more